src/pages/workspace/tags/ImportTagsOptionsPage.tsx

```diff
@@ -136,17 +141,21 @@ function ImportTagsOptionsPage({route}: ImportTagsOptionsPageProps) {
             <TextLink
                 onPress={() => {
                     if (isMultiLevelTags) {
-                        downloadMultiLevelTagsCSV(
-                            policyID,
-                            () => {
-                                setIsDownloadFailureModalVisible(true);
-                            },
-                            hasDependentTags,
+                        close(() =>
+                            downloadMultiLevelTagsCSV(
+                                policyID,
+                                () => {
+                                    setIsDownloadFailureModalVisible(true);
```

Comment:
Oh I thought we only needed to wrap this callback with `close` to prevent overlap modal right 🤔 ?